### ISSUE TITLE
LayoutLM not evaluating all test set

### ISSUE THREAD

**Issue by Lucka-Gianvechio** [microsoft/unilm#615]
Hello! 

I am currently trying to reproduce the results presented at this [Kaggle tutorial](https://www.kaggle.com/urbikn/layoutlm-using-the-sroie-dataset), on how to use the LayoutLM deprecated in the SROIE dataset, but I'm having some problems with the fine-tuning and evaluation steps. I've tried to reproduce almost identicaly every step, but when it gets to the fine-tuning step the following happens:

I am training the model with 10 epochs, but each epoch only marks 1 iteration per epoch. I was expecting something like a bigger number, since i did not change any parameter and the tutorial shows epochs running up to 79 iterations each.

For the evaluationg step my problem is that I have an test set composed with almost 100 images and the script only classifies up to 3 images or less. 

I am using google colab with the GPU environment, and using the LayoutLM model with the deprecated version.
